### DISCUSSION TITLE
Regarding some missing file for drc and lvs run for asap7.

### DISCUSSION THREAD

**Discussion by its-rakib** [The-OpenROAD-Project/OpenROAD-flow-scripts#850] [Q&A]
I am trying to run lvs and drc using klayout. but i couldn't find .lydrc  and .lylvs file which is necessary for the lvs and drc run. it would be great if you can help me with the issues.

**Comment by vijayank88** [comment]
Currently calibre has support to run DRC/LVS check with ASAP7. With KLayout not yet implemented.

**Comment by rovinski** [comment]
There is a community-contributed KLayout enablement [here](https://github.com/laurentc2/ASAP7_for_KLayout) but we can't verify its completeness / accuracy.
